.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+.serverless
```

Comment:
Lets remove these folders from MR (they are commited)

See this thread: https://stackoverflow.com/questions/7527982/applying-gitignore-to-committed-files

Reply:
Done